### DISCUSSION TITLE
Does skywalking 6.x version support OpenAPI specifications

### DISCUSSION THREAD

**Discussion by badabapidas** [apache/skywalking#7666] [Q&A] [answered]
Hi Team,

Need one info regarding OpenAPI specifications. Do skywalking 6.x version support OpenAPI specifications?

It would be great if someone can give some information in this regards.

Bapi

**Comment by wu-sheng** [comment, marked as answer]
No, it doesn't. OpenAPI is recently added by @wankai123 from 8.7.0.

**Reply by badabapidas** [reply]
Thanks @wu-sheng for the reply.
